Makefile

```diff
@@ -3,14 +3,13 @@ CC = gcc
 ECHO = echo
 RM = rm -f
 
-CFLAGS = -Wall -Werror  -ggdb -funroll-loops
```

Comment:
add back

Reply:
idk what funroll-loops is

Reply:
no add back -Werror lil ape, and u need to read it as -f unroll-loops it means if you can remove loops and do the same thing in less instructions like if u were doing a sum loop of all numbers up to n is (n(n-1))/2 or whatever the formula is